modules/nf-core/simpleaf/quant/main.nf

```diff
@@ -0,0 +1,68 @@
+process SIMPLEAF_QUANT {
+    tag "$meta.id"
+    label 'process_high'
```

Comment:
I think we should add process medium, since one of the perks of using simpleaf/alevin_fry is the small memory footprint.